Makefile.dune

```diff
@@ -138,6 +138,21 @@ ireport:
 clean:
 	dune clean
 
+# ci-* targets
+
+CI_PURE_DUNE:=1
+export CI_PURE_DUNE
+include Makefile.ci
+
+# if we do eg "make states ci-foo", ci-foo will wait for states
+# if we just do "make ci-foo" it will just run ci-foo
+# (technically the ci-* targets depend on world but it can be
+# convenient to run them with less than world compiled)
+NON_CI_GOALS:=$(strip $(filter-out ci-%,$(MAKECMDGOALS)))
+ifneq (,$(NON_CI_GOALS))
+$(CI_TARGETS): $(NON_CI_GOALS)
```

Comment:
BTW one might think 
```suggestion
ci-%: $(NON_CI_GOALS)
```
should work but experimentally it doesn't.

The current solution works so no need to look deeper into it ;)